omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -14,7 +14,7 @@ OMERO.server Mac OS X installation walk-through with Homebrew
     the downloads page or build from source.**
 
 These instructions are implemented in a series of `automated scripts
-<https://github.com/ome/omero-install/tree/master/homebrew>`_ which
+<https://github.com/ome/omero-install/tree/develop/osx>`_ which
```

Comment:
this link is failing the jenkins build - https://ci.openmicroscopy.org/job/OMERO-DEV-merge-docs/135/warnings3Result/